src/linux/osd.c

```diff
@@ -257,3 +257,49 @@ size_t ofi_ifaddr_get_speed(struct ifaddrs *ifa)
 }
 
 #endif /* HAVE_ETHTOOL */
+
+int ofi_ifaddr_get_mtu(const struct ifaddrs *ifa)
+{
+	FILE *fd;
```

Comment:
Use `fp` instead. `fd` is usually used for the integer file descriptor returned from `open`.